projects/addon-doc/components/main/main.style.less

```diff
@@ -148,7 +148,6 @@ tui-doc-code {
     .hljs-comment,
     .hljs-quote {
         color: var(--tui-text-02);
-        font-style: italic;
```

Comment:
Why not italic on comments?

Reply:
<img width="253" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/3e4209c0-ac34-410e-8970-bc20c2a9884b">
in a familiar IDE

Reply:
fixed